lib/openscap_parser/profile.rb

```diff
@@ -20,8 +20,13 @@ def description
     end
 
     def selected_rule_ids
-      @selected_rule_ids ||= @parsed_xml.xpath("select[@selected='true']/@idref") &&
-        @parsed_xml.xpath("select[@selected='true']/@idref").map(&:text)
+      @selected_rule_ids ||= @parsed_xml.xpath("select[@selected='true'][contains(@idref, 'xccdf_org.ssgproject.content_rule_')]/@idref") &&
+        @parsed_xml.xpath("select[@selected='true'][contains(@idref, 'xccdf_org.ssgproject.content_rule_')]/@idref").map(&:text)
```

Comment:
The `contains` here is a good start, but maybe I'd go with a regular expression if nokogiri allows that. Maybe then we can just go with a positive+negative lookahead and match the first `_group_` prefix that doesn't preceeds with a `rule` and also the first `_rule_` that doesn't preceeds with a `group`.

Reply:
Oh yeah, that could work! I'll work on this.